v2/workloadapi/addr_windows_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build windows
+// +build windows
+
+package workloadapi
+
+func validateAddressCasesOS() []validateAddressCase {
+	return []validateAddressCase{
+		{
+			addr: "npipe:pipeName",
+			err:  "",
+		},
+		{
+			addr: "npipe:pipe/name",
+			err:  "",
+		},
+		{
+			addr: "npipe:pipe\\name",
```

Comment:
Should the backslash be allowed? From CreateNamedPipe:

> The pipename part of the name can include any character other than a backslash, including numbers and special characters. The entire pipe name string can be up to 256 characters long. Pipe names are not case sensitive.

Reply:
That's interesting. We have been using backslashes in the pipe name in SPIRE (we have set the default named pipe name for the Workload API named pipe as `\spire-agent\public\api`).

Looking at the code in the go-winio library, I see that instead of the CreateNamedPipe function, the (undocumented?) NtCreateNamedPipeFile function is used: https://github.com/microsoft/go-winio/blob/master/pipe.go#L338

I'm thinking that's why we didn't run into this limitation when creating the pipes in SPIRE. It seems that the NtCreateNamedPipeFile function allows the use of backslashes (and also the pipe name string can be larger than 256 long).

We can probably remove this test case, but I'm thinking that we may not perform additional validations to check that the pipe name conforms with the CreateNamedPipe function. What do you think?